Chef/Vagrantfile

```diff
@@ -386,181 +386,188 @@ Vagrant.configure('2') do |config|
+  config.vm.provision 'chef_zero' do |chef|
+    chef.cookbooks_path = 'Cookbooks'
+    chef.data_bags_path = 'DataBags'
+    chef.nodes_path = 'Nodes'
+    chef.roles_path = 'Roles'
+    chef.channel = 'stable'
+    chef.version = '13.8.5'
+    chef.json = {
+      'smb_domain' => network_share_smb_domain,
+      'smb_username' => network_share_smb_username,
+      'smb_password' => network_share_smb_password,
+      'smtp_server' => relativity_smtp_server,
+      'smtp_port' => relativity_smtp_port,
+      'email_from' => relativity_email_from,
+      'email_to' => relativity_email_to,
+    }
+    chef.add_recipe('Relativity::post_relativity_instance_setting_update_developer_mode')
+    chef.add_recipe('Relativity::post_relativity_reset_iis')
+    chef.add_recipe('Relativity::post_relativity_update_kcura_services_startup_type')
+    chef.add_recipe('Relativity::post_relativity_create_workspace')
+    # chef.add_recipe('Relativity::post_relativity_create_data_sampler_config_file') # (OLD)
+    # chef.add_recipe('Relativity::post_relativity_push_resource_file') # (OLD)
+    # chef.add_recipe('Relativity::post_relativity_install_rap_files') # (OLD)
+    # chef.add_recipe('Relativity::post_relativity_create_rap_agents') # (OLD) agent api discontinued. so failing. need to be fixed.
+    chef.add_recipe('Relativity::post_relativity_install_smoke_test_rap') # (Aaron)
+  end
 
-  # # Cleanup VM
-  # config.vm.provision 'chef_zero' do |chef|
-  #   chef.cookbooks_path = 'Cookbooks'
-  #   chef.data_bags_path = 'DataBags'
-  #   chef.nodes_path = 'Nodes'
-  #   chef.roles_path = 'Roles'
-  #   chef.channel = 'stable'
-  #   chef.version = '13.8.5'
-  #   chef.json = {
-  #     'smb_domain' => network_share_smb_domain,
-  #     'smb_username' => network_share_smb_username,
-  #     'smb_password' => network_share_smb_password,
-  #     'smtp_server' => relativity_smtp_server,
-  #     'smtp_port' => relativity_smtp_port,
-  #     'email_from' => relativity_email_from,
-  #     'email_to' => relativity_email_to,
-  #   }
-  #   chef.add_recipe('Relativity::post_relativity_cleanup')
-  #   chef.add_recipe('Relativity::post_relativity_write_script_result_success')
-  # end
+  # Cleanup VM
+  config.vm.provision 'chef_zero' do |chef|
+    chef.cookbooks_path = 'Cookbooks'
+    chef.data_bags_path = 'DataBags'
+    chef.nodes_path = 'Nodes'
+    chef.roles_path = 'Roles'
+    chef.channel = 'stable'
+    chef.version = '13.8.5'
+    chef.json = {
+      'smb_domain' => network_share_smb_domain,
+      'smb_username' => network_share_smb_username,
+      'smb_password' => network_share_smb_password,
+      'smtp_server' => relativity_smtp_server,
+      'smtp_port' => relativity_smtp_port,
+      'email_from' => relativity_email_from,
+      'email_to' => relativity_email_to,
+    }
+    chef.add_recipe('Relativity::post_relativity_cleanup')
+    chef.add_recipe('Relativity::post_relativity_delete_errors_from_errors_tab') # (Aaron)

```

Comment:
we need to re-arrange some of the recipe's order. We can do it when I come back to office.